lib/diagnostic.rb

```diff
@@ -7,36 +7,38 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    array = list.split('')
+    array.sort! { |a, b| b.length <=> a.length }
+    array[0]
```

Comment:
Too much code! Ruby likes to do things as quickly as possible like so:

`list.max_by(&:length)`